tools/clang/test/HLSLFileCheck/passes/llvm/gvn/GVN_readfirstlane.hlsl

```diff
@@ -0,0 +1,17 @@
+// RUN: %dxc -E main -T ps_6_0 %s  | FileCheck %s
```

Comment:
I don't think I can comment on the broader fallout of this change, but I thought I'd point out that this test passes even without this change. dx.op.waveReadLaneFirst is not eliminated

Reply:
I simplified local test before commit :(
Will revert it back to longer test.